_data/sites.json

```diff
@@ -14096,6 +14096,16 @@
         ]
     },
 
+    {
+        "name": "Omnipilot",
+        "url": "https://omnipilot.ai/account",
+        "difficulty": "easy",
+        "notes": "On your account page, click on 'Delete Account' then confirm.",
+        "notes_pt_br": "Nas preferências do seu perfil, clique em 'Delete Account' e depois confirme.",
+        "domains": [
+            "omnipilot.ai"
+        ]
+    },
```

Comment:
```suggestion
        "domains": [
            "omnipilot.ai"
        ]
    },

```

Reply:
Argh, I cannot at this line break on the gh mobile app because it thinks there's no diff here. Can you commit the proposed suggestion, please?